backend/trustyserver/embed/client.go

```diff
@@ -0,0 +1,18 @@
+package embed
+
+import (
+	"context"
+
+	"github.com/go-phorce/trusty/backend/trustyserver"
+	"github.com/go-phorce/trusty/client"
+	"github.com/go-phorce/trusty/client/proxy"
+)
+
+// NewClient returns client.Client for running TrustyServer
+func NewClient(s *trustyserver.TrustyServer) *client.Client {
```

Comment:
More of a philosophical comment. See by having `github.com/go-phorce/trusty/client` we closely tie embed package to the client package so we can't later move embed package without dragging client package with it. Just for that reason i feel client package should be defined under embed folder (backend/trustyserver/embed/client/...). The callers can choose to either use client provided by embed server or write a custom one by their own.

Reply:
correct. this package can't be refactored out as it's tied to `*trustyserver.TrustyServer` to be used inside the running instance or embedded server.